src/components/Cauldrons.vue

```diff
@@ -0,0 +1,197 @@
+        :data-bs-target="'#' + tab"
+        type="button"
+        role="tab"
+        aria-controls="profile"
+        :aria-selected="tab === 'Orange'"
+        :style="{background: colorTo(tab, 'Hex')}"
+        v-on:click="setGroup(tab)"
+      >
+        {{colorTo(tab, "Name")}} Cauldron
+      </button>
+    </li>
+  </ul>
+  <div class="d-flex flex-column alchemy-wrapper mt-3">
+    <!-- Discount table -->
+    <table class="border-top border-bottom text-light">
+      <tr class="border-bottom">
+        <th class="centered"> Cost Reductions </th>
+      </tr>
+      <tr :style="{display: 'flex', justifyContent: 'space-between'}">
+        <th class="quarter centered">
+          Cauldron Level: <br>
+          <input class="columnSpan"
+            v-model="cauldRedux"
+            type="number" :min="0"
+            @change="handleDiscountRecalc()"/>
+        </th>
+        <th class="quarter centered">
+          Bargain tag Level: <br>
+          <input class="columnSpan"
+            v-model="bargainLvl"
+            type="number" :min="0"
+            @change="handleDiscountRecalc()"/>
+        </th>
+       <th class="quarter centered spaced"
+          v-for="d in ['XII Bubble', 'Undeveloped cost \nbubble and Iron bar vial', 'Total']"
+          :key="d">
+          {{d}}
+        </th>
+      </tr>
+      <tr :style="{display: 'flex'}">
+        <td class="quarter centered"
+          v-for="(d, index) in discount" :key="index">
+          {{d.toFixed(2)}}%
+        </td>
+      </tr>
+    </table>
+    <!-- Data table -->
+    <table class="text-light">
+      <tr class="border-bottom">
+        <th class="centered columnSpan"
+          v-for="(item, index) in headers" :key="index">
+          {{item}}
+        </th>
+        <th>Materials</th>
+      </tr>
+      <tr class="alter-color"
+        v-for="n in 15" :key="n">
+        <AlchemyRow @customChange="handleDiscountRecalc()" :group="activeGroup" :idx="amountBubbles-n" :discount="discount" />
+      </tr>
+    </table>
+  </div>
+</template>
+
+<script lang="ts">
+  import { computed, defineComponent, ref } from "vue";
+  import { useState} from "~/State";
+  import AlchemyRow from "~/components/AlchemyRow.vue";
+  import { Color, Alch } from "~/composables/Alchemy";
+
+
+  export default defineComponent({
+      name: "Cauldrons",
+      components: {
+          AlchemyRow,
+      },
+      setup() {
+          const state = useState();
+          const alchemy = computed({
+            get: () => state.value.alchemy,
+            set: (value) => (state.value.alchemy = value),
+          });
+
+          const colors: Color[] = ["Orange", "Green", "Purple", "Yellow"];
+          const colorScheme = ["#ff9420", "#47db5a", "#cb79e3", "#edc300"];
+          const naming = ["Power", "Quicc", "High IQ", "Kazam"];
+
+          const headers = ["Icon", "Level", "Goal", "Effect"];
+          const amountBubbles = 15;
+
+          const activeGroup = ref("Orange"); 
```

Comment:
```suggestion
          const activeGroup = ref<Color>("Orange"); 
```
So we know that activeGroup is always a color.